src/splitInteger.test.js

```diff
@@ -8,12 +8,40 @@ describe(`Function 'splitInteger':`, () => {
   });
 
   it(`should return an array`, () => {
-
+    expect(splitInteger(8, 4))
+      .toBeInstanceOf(Array);
   });
 
   it(`should return an array with 'num' if parts = 1`, () => {
+    const expectNum = 3;
 
+    expect(splitInteger(expectNum, 1)).toEqual([3]);
   });
-  
-  // write more tests here
+
+  it(`should return an array of integers`, () => {
+    const res = (splitInteger(20, 3));
+
+    for (const check of res) {
+      expect(typeof check).toBe('number');
+    }
+  });
+
+  it(`should return an array which length is equal to 'parts' value`, () => {
+    expect(splitInteger(20, 3).length).toBe(3);
+  });
```

Comment:
You can also use [toHaveLength](https://jestjs.io/docs/expect#tohavelengthnumber) here